server/routes/routes.js

```diff
@@ -24,4 +28,64 @@ router.get("/activities", async (req, res) => {
     res.status(500).send("Failed to fetch data");
   }
 });
+
+router.post("/create-user", async (req, res) => {
+  const { id, email, name } = req.body;
+
+  try {
+    const user = await prisma.user.create({
+      data: {
+        id: id,
+        email: email,
+        name: name || "User",
+      },
+    });
+    res.status(201).json(user);
+  } catch (error) {
+    if (error.code === "P2002") {
+      res.status(409).send("A user with this email already exists.");
```

Comment:
try to send json for error responses as well. Would be easy to parse in frontend